### PR TITLE
Add new change image to IM-Dashboard

### PR DESCRIPTION
**Pull request by micafer**
# Summary

Add new IM-Dasboard functionality that enables the user to change the base image to new nodes.

### CODE REVIEW

File: content/en/users/compute/orchestration/im/dashboard/_index.md [hunk 1]
```diff
@@ -135,7 +135,11 @@ infrastructures:
 - Add nodes: The Add nodes action enables to add new VMs to the users'
   deployment. It will show the list of the different types of nodes currently
   deployed in the infrastructure and the user must set the number of nodes of
-  each type he wants to deploy.
+  each type he wants to deploy. It will also show the image selected to create
+  the VMs (if available). In the dropdown list the user can change the base
+  image, in case that you want to use a new one on the new deployed nodes, or
+  the old one has been removed and you need to select a new one to be able to
+  deploy them.
```
Comment (gwarf) [inline content/en/users/compute/orchestration/im/dashboard/_index.md:142]: There is a mix of using `he/the user` and `you`, it should be aligned, ideally to use the second person as mentioned on https://docs.egi.eu/about/contributing/style/#write-in-second-person

Comment (micafer) [reply content/en/users/compute/orchestration/im/dashboard/_index.md]: As in the rest of the document third person is used I alined it to use it.

Comment (gwarf) [reply content/en/users/compute/orchestration/im/dashboard/_index.md]: OK, but let's aim at having the full document be moved to 2nd person at some point, as it's the expected way.

File: content/en/users/compute/orchestration/im/dashboard/_index.md [hunk 2]
```diff
@@ -135,7 +135,9 @@ infrastructures:
 - Add nodes: The Add nodes action enables to add new VMs to the users'
   deployment. It will show the list of the different types of nodes currently
   deployed in the infrastructure and the user must set the number of nodes of
-  each type he wants to deploy.
+  each type he wants to deploy. It will also show the image selected to create
+  the VMs (if available). In the dropdown list the user can change the base
+  image, to use a new one or because the old one has been removed.
```
Comment (enolfc) [inline content/en/users/compute/orchestration/im/dashboard/_index.md:140]: This sentence reads a bit weird. Could you clarify?

**Review by enolfc — CHANGES_REQUESTED** [review]
Thanks @micafer. Just one minor comment.

File: content/en/users/compute/orchestration/im/dashboard/_index.md (hunk 2 above)
Comment (micafer) [reply content/en/users/compute/orchestration/im/dashboard/_index.md]: Is it clear now?

**Review by enolfc — APPROVED** [review]
LGTM!

**Pull request merged**